src/main/java/com/amazonaws/encryptionsdk/AwsCrypto.java

```diff
@@ -281,7 +281,12 @@ public long estimateCiphertextSize(
     /**
      * Calls {@link #encryptData(MasterKeyProvider, byte[], Map)} on the UTF-8 encoded bytes of
      * {@code plaintext} and base64 encodes the result.
+     * @deprecated Use the {@link #encryptData(MasterKeyProvider, byte[], Map)} and
+     * {@link #decryptData(MasterKeyProvider, byte[])} APIs instead. {@code encryptString} is incompatible with the
```

Comment:
I would not say that they are incompatible without clarification. Maybe "because the the output must be base64-decoded to be compatible with other AWS Encryption SDK implementations"?

Reply:
Works for me, I am but a deprecation-shaped vessel for the wordsmithing that you guys think make sense.

Reply:
Because it returns Base64-encoded ciphertext, this API is not fully compatible with other AWS Encryption SDK language implementations.

Reply:
> Because these APIs return Base64-encoded output, they are not compatible with other AWS Encryption SDK language implementations.

IMO this still conveys the wrong message.
What I would like to make sure we're communicating is that they *are* compatible, *but* only once you peal away an extra layer of encoding that this method adds.

Reply:
After some noodling, I've tried again to be clear and concise in explaining that this is deprecated, and why. When you have a second, please let me know what you think of the new wording.